tic-tac-toe/src/components/control-pannel/control-pannel.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import './control-pannel.css';
+
+const GameModePannel = ({activeGameMode, onChangeMode, outerClassName}) => {
+    const innerClassName = 'mode-panel';
+    const className = `${outerClassName} ${innerClassName}`
```

Comment:
It's overkill here. Just add a className to your component.

Reply:
I fix this.
```suggestion
const GameModePannel = ({activeGameMode, onChangeMode}) => {
  const buttons = buttonsInfo.map(({name, label}) => {
      const isActive = (name === activeGameMode);
      let btnClassName = classes.Button;
      btnClassName += (isActive) ? ` ${classes.ActiveButton}` : '';
      return (
```